packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -473,6 +480,35 @@ export const RemoteServerFactory = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupRemoveBasePathFromPathMiddleware(app) {
```

Comment:
This middleware that removes the base path is the magic that makes base paths work on a local environment.

On MRT, this also serves to correct as much as possible any requests that contain the base path if they are sent to lambda.

Reply:
The current setup removes the base path from every URL that makes it through CDN to the server.

This might be problematic if the envBasePath is the same as the site alias.
Suppose our base path is /us:
We'll want to remove the base path from /us/mobify
But not /us/en-US/product

Reply:
I could apply this removal only to a set of urls like /mobify or whatever routes have been defined in express but that might be overcomplicating the solution?

Reply:
I proceeded with refining the middleware so that it only removes the base path from /mobify routes or routes defined in express